ppafm/cli/gui/ppafm_gui.py

```diff
@@ -246,11 +246,16 @@ def updateRotation(self):
         if self.verbose > 0: print('updateRotation', a, self.rot)
         self.update()
 
-    def updateParams(self):
+    def updateParams(self, preset_none=True):
         '''Get parameter values from input fields and update'''
 
         if self.xyzs is None: return
 
+        if preset_none:
+            self.slPreset.blockSignals(True)
```

Comment:
What is this? I undrestand this is some locking / synchronization due to parallel access, but woudl be nice to explain it? (e.g. docstring?) It is something general, or your own solution?

Reply:
I'm not that well-versed with pyQt, so there could be some more elegant solution. The reason why those blocks are there is because otherwise it would execute the function bound to the widget (selection box here).

Whenever we connect a function to the widget like 
```python
self.slPreset.currentIndexChanged.connect(self.applyPreset)
```
then all changes to the widget will cause the function to be called, including changes made programmatically. In this case I just wanted to change the display value of the widget without calling the function, so I temporarily block the signals and then enable them again after the change has been made.

Reply:
please try to mention it briefly in the code. This is exactly one of those things which not obvious because it is `non-local` and therefore difficult to figure out

Reply:
I created another PR for this: #140.